src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/notifications/TestData.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.notifications;
+
+import uk.gov.hmcts.reform.finrem.caseorchestration.FinremCaseDetailsBuilderFactory;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.NottinghamCourt;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.Region;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.RegionMidlandsFrc;
+
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CCDConfigConstant.NOTTINGHAM;
+
+public abstract class TestData implements TestConstants {
```

Comment:
This doesn't seem right to me logically. TestConstants doesn't seem to be an interface. From an object-oriented point of view you'd use composition rather than inheritance to utilise TestConstants in TestData.